Lib/multiprocessing/context.py

```diff
@@ -46,6 +51,18 @@ def cpu_count(self):
         else:
             return num
 
+    @property
+    def Process(self):
+        if self._reducer is None:
+            # Ensure backward compatibility by returning a class when no
+            # custom reducer was specified
+            return self._Process
+        else:
+            return self.process
+
+    def process(self, *args, **kwargs):
```

Comment:
```suggestion
    def _process(self, *args, **kwargs):
```
This attribute should not be public.